.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "${workspaceRoot}/venv/bin/python"
+}
```

Comment:
tisk tisk: `git add .`

Reply:
Haha. The git problems I caused for myself on this branch are beyond `add .` unfortunately, but this won't be in the final version.